### PR TITLE
Fix browser routing

### PR DESCRIPTION
**Pull request by jyothishnt**
## Type

- [x] Bug fix
- [ ] New feature
- [ ] Improvement to existing feature

## Related JIRA Issue(s)
[ENSWBSITES-625](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-625)

## Description
Links from homepage to other apps always been taken to humans. Now we pass the genome id and it is fixed.

## Views affected
Homepage

- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

## Possible complications
Nothing that I am aware

### CODE REVIEW

File: src/ensembl/src/content/app/browser/browserActions.ts [hunk 1]
```diff
@@ -363,7 +363,6 @@ export const changeFocusObject = (
   }
 
   dispatch(updatePreviouslyViewedObjectsAndSave());
-
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/browserActions.ts:366]: could you revert this change please?

Comment (azangru) [reply src/ensembl/src/content/app/browser/browserActions.ts:366]: yep, the removal of the line between dispatch and browserMessagingService.

In fact, could you run `git checkout dev <path to src/content/app/browser/browserActions.ts>`? It will restore the file to its state in dev branch; and you'll be able to commit the restored file.

Comment (jyothishnt) [reply src/ensembl/src/content/app/browser/browserActions.ts:366]: yup actually there was no change in that file LOL

Comment (jyothishnt) [reply src/ensembl/src/content/app/browser/browserActions.ts]: Done!
